src/videojs-hls.js

```diff
@@ -671,7 +678,6 @@ videojs.Hls.prototype.loadSegment = function(segmentUri, offset) {
       offset: offset,
       bytes: new Uint8Array(this.response)
     });
-    player.trigger('progress');
```

Comment:
progress events are already firing from videojs because the bufferedPercent is changing.  Since all the other changes in this pull has been moved into mediaSources I opened a separate pull request to remove just this event #252.  Firing progress here was sometimes causing the buffered region in the player to misreport.